logger/logger.go

```diff
@@ -0,0 +1,35 @@
+package logger
+
+import (
+	"strings"
+
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+type LoggerSettings struct {
+	EnableConsole bool
+	ConsoleJson   bool
+	ConsoleLevel  string
+	EnableFile    bool
+	FileJson      bool
+	FileLevel     string
+	FileLocation  string
+}
+
+func InitLogger(logSettings *LoggerSettings) {
```

Comment:
Since you made the `logger` package (great idea) I suggest we rename this to just `Init`. Then calling `logger.Init()` will look better than `logger.InitLogger()`.